pipeline/scraper/PSQL.py

```diff
@@ -4,7 +4,8 @@
 import re
 
 INSERT_TO_APPLICATION_SQL = "INSERT INTO applications (url, name, followers, retrieved, hash) VALUES (%s, %s, %s, %s, %s) ON CONFLICT ON CONSTRAINT unique_url DO UPDATE SET (url, name, followers, retrieved, hash) = (EXCLUDED.url, EXCLUDED.name, EXCLUDED.followers, EXCLUDED.retrieved, EXCLUDED.hash) RETURNING id;"
-INSERT_TO_PACKAGES_SQL = "INSERT INTO packages (name, downloads_last_month, categories, modified, retrieved) VALUES (%s, %s, %s, %s, %s) ON CONFLICT(name) DO UPDATE SET (name, downloads_last_month, categories, modified) = (EXCLUDED.name, EXCLUDED.downloads_last_month, EXCLUDED.categories, EXCLUDED.modified) RETURNING id;"
+INSERT_TO_PACKAGES_SQL = "INSERT INTO packages (name, retrieved) VALUES (%s, %s) ON CONFLICT(name) DO UPDATE SET (name, retrieved) = (EXCLUDED.name, EXCLUDED.retrieved) RETURNING id;"
+UPDATE_PACKAGE_METADATA_SQL = "UPDATE packages SET downloads_last_month = %s, categories = %s, modified = %s WHERE name = %s;"
```

Comment:
would be nice to break into more lines these long ones (as SQL clauses go)

Reply:
Fixed